Sources/Plasma/Apps/plClient/CMakeLists.txt

```diff
@@ -116,6 +116,10 @@ elseif(APPLE)
         Mac-Cocoa/PLSLoginWindowController.xib
         Mac-Cocoa/PLSPatcherWindowController.xib
     )
+    list(APPEND plClient_SHADERTARGETS
+        pfMetalPipelineShadersMSL21
+        pfMetalPipelineShadersMSL23
+    )
```

Comment:
This should probably be pulled in transitively from `pfMetalPipeline` target.

Reply:
I moved what I could but this specific part (the XCODE_EMBED_RESOURCES target property) is not transitive.